### PR TITLE
Craig 74

### PR DESCRIPTION
**Pull request by jvallexm**

### CODE REVIEW

File: storybook/src/stories/Forms/ClusterForm.stories.js [hunk 1]
```diff
@@ -167,32 +192,33 @@ const ClusterFormStory = () => {
...
         kube_version: "4.10.52_openshift (Default)",
         update_all_workers: false,
-        worker_pools: [],
+        worker_pools: [
+          {
+            entitlement: "cloud_pak",
+            cluster: "workload",
+            flavor: "bx2.16x64",
+            name: "logging-pool",
+            resource_group: "slz-workload-rg",
```
Comment (isaaceichelberger) [inline storybook/src/stories/Forms/ClusterForm.stories.js:216]: remove slz

File: storybook/src/stories/Forms/ObjectStorage/ObjectStorageForm.stories.js [hunk 2]
```diff
@@ -116,21 +116,53 @@ const ObjectStorageFormStory = () => {
...
+        ],
+        name: "atracker-cos",
+        plan: "standard",
+        resource_group: "slz-service-rg",
         use_data: false,
-        resource_group: "rg1",
-        random_suffix: true,
-        kms: "kms-1",
+        use_random_suffix: false,
+        kms: "slz-kms",
```
Comment (isaaceichelberger) [inline storybook/src/stories/Forms/ObjectStorage/ObjectStorageForm.stories.js:140]: remove slz references

File: src/components/forms/ClusterForm.js [hunk 3]
```diff
@@ -5,15 +5,18 @@ import {
 } from "../component-utils";
 import { handleClusterInputChange } from "../../lib";
 import { IcseNameInput, IcseToggle } from "../Inputs";
-import { IcseHeading, IcseFormGroup, RenderForm } from "../Utils";
+import { IcseHeading, IcseFormGroup } from "../Utils";
+import WorkerPoolForm from "./WorkerPoolForm";
```
Comment (isaaceichelberger) [inline src/components/forms/ClusterForm.js:9]: missing new private endpoint toggle/prop

File: src/components/forms/WorkerPoolForm.js [hunk 4]
```diff
@@ -40,12 +38,16 @@ class WorkerPoolForm extends Component {
...
+    let pool = { ...this.state };
     pool.subnets = event.selectedItems;
     this.setState({ pool });
   }
 
   render() {
+    if(this.props.isModal) {
+      console.log(this.state)
+      console.log(this.props)
+    }
```
Comment (isaaceichelberger) [inline src/components/forms/WorkerPoolForm.js:50]: remove

**Review by hjweds — APPROVED** [review]

**Review by isaaceichelberger — APPROVED** [review]

**Pull request merged**